GRPCServiceA/pkg/user/service.go

```diff
@@ -0,0 +1,42 @@
+package user
+
+import (
+	"context"
+
+	ent "github.com/coding-kiko/GoKit-Project-Bootcamp/GRPCServiceA/pkg/entities"
+	"github.com/go-kit/kit/log"
+)
+
+type service struct {
```

Comment:
naming the implementation as the interface can be a confusion, try to add some description, additional when you try to add other service It will be useful to distinct